backend/inspirehep/records/api/mixins.py

```diff
@@ -379,17 +380,13 @@ def get_modified_institutions_uuids(self):
         changed_deleted_status = self.get("deleted", False) ^ prev_version.get(
             "deleted", False
         )
-        pids_latest = self.get_linked_pids_from_field("authors.affiliations.record")
+        pids_latest = list(self.linked_institutions_pids)
```

Comment:
are all these `list` really needed?

Reply:
This one is needed because of ```get_records_ids_by_pids(pids_latest)``` which has ``` pids[max_batch * batch_no : max_batch * (batch_no + 1)]```